src/reputation/management/commands/complete_pending_payout.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from purchase.models import Balance
+from reputation.lib import (
+    PendingWithdrawal
+)
+from reputation.models import Withdrawal
+from django.contrib.contenttypes.models import ContentType
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        withdrawals_to_complete = [1868, 1864]
```

Comment:
It seems like these transactions are hard-coded which will give us a one-time usage out of this command. Does it make sense to accept the transactions as an arg instead?

Reply:
I think we should just change this everytime we need to run it. An arg would be OK too if we see us using this a lot.

Reply:
Sounds good